diffmatchpatch/diff_test.go

```diff
@@ -1456,6 +1454,7 @@ func TestDiffMainWithCheckLines(t *testing.T) {
 }
 
 func TestMassiveRuneDiffConversion(t *testing.T) {
+	t.Skip()
```

Comment:
I realized I skipped this test by accident and forgot to revert this. This test is failing and I will try to fix it. IT seems my implementation with handling invalid unicode codepoints is naive and does not work all the time. I will try to fix it and update this PR.

Reply:
I fixed integer to rune conversion logic and this test is working now.